homeWork2/getDetailedInfoPokemon.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+async function getDetailedInfoPokemon(pokemon) {
```

Comment:
1. getPokemonDetails shorter and would be enough
2. pokemonName would be better, because from quick look it sounds like pokemon is an object